designs/virtualization/4.x/Templates/Templates.md

```diff
@@ -0,0 +1,49 @@
+# Templates
+
+## Virtual machines and VM templates are located under Virtualization
+
+Removing the tab ‘Virtual Machine Templates’ from the side navigation and combining it with ‘Virtual Machines’, so both are located under ‘Virtualization’.
+This PR will focus on how users can create a VM when they choose a to create a VM from a template.
+
+### Virtual Machine Templates Tab
+
+From the Virtualization left nav item the user goes to the Virtual machine templates tab where they can see the full list of templates.
+From this point, we suggest 2 options to create a VM:
+
+- Create VM via an action from the kebab menu
+
+![TemplatesTabOp1](img/TemplatesTabsOp1.png)
+
+- Create VM via an action from the kebab menu and also allow an action link
+
+![TemplatesTabOp2](img/TemplatesTabsOp2.png)
+
+Both options will lead the user to the general step of the wizard where they will need to provide a name for the VM they want to create.
```

Comment:
Are we still deciding on one way or the other? I like the 2nd option where the Create is surfaced all of the time if we don't think it's too repetitive :)

Reply:
@lizsurette I rephrased the sentence so it will be clearer that we offer 2 options for 'create' and not one way or another.

Reply:
Thanks!